### PR TITLE
PaneHeader Storybook

### PR DESCRIPTION
**Pull request by epeach**
For the comment box feature, I made a lot of small changes to components in a way that is challenging to test. Caley suggested adding stories as a means of checking expected output for these components.

This PR adds stories for PaneHeader (and PaneSection and PaneButton).

This PR also removes the commonStyle associated with the prop removed from PaneHeader in this PR: https://github.com/code-dot-org/code-dot-org/pull/23801

This PR adds styling to PaneHeader in teacher-only mode that emulates the PaneButton styles for focus/un-focus when not in teacher-only mode.

Standard header with sections and button. Second header does not have focus.

Teacher-only header with sections and button. Second header does not have focus.
screenshot from 2018-07-27 16-35-37

Teacher-only header displaying both LTR and RTL buttons.
screenshot from 2018-07-27 16-35-52

### CODE REVIEW

File: apps/src/commonStyles.js [hunk 1]
```diff
@@ -17,10 +17,6 @@ commonStyles.purpleHeader = {
   userSelect: 'none',
 };
 
-commonStyles.purpleHeaderReadOnly = {
```
Comment (epeach) [inline apps/src/commonStyles.js:20]: Read-only prop removed here: https://github.com/code-dot-org/code-dot-org/pull/23801

File: apps/src/templates/PaneHeader.jsx [hunk 2]
```diff
@@ -162,8 +163,9 @@ export const PaneButton = Radium(function (props) {
     >
       <span style={styles.headerButtonSpan}>
         {props.hiddenImage}
-        <i className={props.iconClass} style={iconStyle}/>
+        {!props.isRtl && <i className={props.iconClass} style={iconStyle}/>}
```
Comment (epeach) [inline apps/src/templates/PaneHeader.jsx:166]: @Erin007 Do you know of a better way to do this?

File: apps/src/templates/PaneHeader.story.jsx [hunk 3]
```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+import PaneHeader, {PaneSection, PaneButton} from './PaneHeader';
+
+const styles = {
+  header: {
```
Comment (Erin007) [inline apps/src/templates/PaneHeader.story.jsx:5]: Why do we need styles in the Story? Should they be baked into the components or are these styles already part of the page on which these components are used?

**Comment by Erin007** [comment]
Code looks good so far. Screenshots for context, please?

File: apps/src/templates/PaneHeader.jsx (hunk 2 above)
Comment (Erin007) [reply apps/src/templates/PaneHeader.jsx:166]: What you have is consistent with how we do this in other places, such as https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/templates/LargeChevronLink.jsx.

File: apps/src/templates/PaneHeader.story.jsx (hunk 3 above)
Comment (epeach) [reply apps/src/templates/PaneHeader.story.jsx:5]: The styles are usually baked into the page or added as need to display titles, buttons, icons, wherever requested in the spec. I included these styles to make it easier to see the multiple buttons and sections that I've added to the headers to reduce the amount of duplicating needed to show all the different combinations.

**Comment by epeach** [comment]
@Erin007 PTAL

**Pull request merged**